META.json

```diff
@@ -1,6 +1,7 @@
 {
    "name": "ajbool",
    "abstract": "Three valued boolean with pseudo-NULL for primary keys",
+   "author": "adjustgmbh",
    "version": "0.0.2",
    "maintainer": [ "Chris Travers <chris.travers@adjust.com>", "Manuel Kniep <rapimo@adjust.com>" ],
```

Comment:
You want to change this too.  I don't imagine Manuel is much of a maintainer anymore and I certainly am not,

Reply:
Yes, especially after talking to Arthur, I don't think there's no maintener specially dedicated to it now.